mo_gymnasium/envs/mujoco/reacher.py

```diff
@@ -0,0 +1,77 @@
+from os import path
+
+import numpy as np
+from gymnasium import utils
+from gymnasium.envs.mujoco import MujocoEnv
+from gymnasium.envs.mujoco.reacher_v4 import ReacherEnv
+from gymnasium.spaces import Box, Discrete
+
+
+DEFAULT_CAMERA_CONFIG = {"trackbodyid": 0}
+
+
+class MOReacherEnv(ReacherEnv):
```

Comment:
Could you add pydoc with some high level explanation of what the env is about?

Reply:
I'll address this in a new PR to solve #24